services/web/src/utils/analytics/README.md

```diff
@@ -0,0 +1,180 @@
+# GTM Tracking
+
+- [Intro](#intro)
+- [Setup](#setup)
+- [Environments](#environments)
+- [Pageviews](#pageviews)
+- [Custom Events](#custom-events)
+  - [Triggers and Tags](#triggers-and-tags)
+  - [Variables](#variables)
+  - [Interaction Events](#interaction-events)
+    - [Clicks](#clicks)
+    - [Element Visibility](#element-visibility)
+    - [Other Interactions](#other-interactions)
+  - [Data Events](#data-events)
+    - [Request Events](#request-events)
+    - [Session Events](#session-events)
+- [User ID Tracking](#user-id-tracking)
+- [Debugging](#debugging)
+
+## Intro
+
+Google Tag Manager is an advanced tool that enables complex tracking with minimal code changes.
```

Comment:
Does "plain" Google Analytics also get enabled with this? If so let's mention that for idiots like me

Reply:
It needs to be set up in GTM but the readme goes into that (and assumes GA for most projects)... updated the readme to make it clearer